include/circt/Dialect/FIRRTL/FIRRTLAnnotations.h

```diff
@@ -68,6 +68,9 @@ class AnnotationSet {
   forPort(Operation *module, size_t portNo,
           SmallVectorImpl<NamedAttribute> &otherAttributes);
 
+  /// Get an annotation set for the specified value.
+  static AnnotationSet getAnnotationSet(Value v);
```

Comment:
The naming seems verbose to me. However, I don't know what is a good naming convention. `get` seems to align better with my understanding of MLIR conventions.

```suggestion
  /// Get an annotation set for the specified value.
  static AnnotationSet get(Value v);
```

Reply:
Thanks, updated the name.